tfe/resource_tfe_workspace.go

```diff
@@ -379,7 +397,8 @@ func resourceTFEWorkspaceUpdate(d *schema.ResourceData, meta interface{}) error
 		d.HasChange("allow_destroy_plan") || d.HasChange("speculative_enabled") ||
 		d.HasChange("operations") || d.HasChange("execution_mode") ||
 		d.HasChange("description") || d.HasChange("agent_pool_id") ||
-		d.HasChange("global_remote_state") || d.HasChange("structured_run_output_enabled") {
+		d.HasChange("global_remote_state") || d.HasChange("structured_run_output_enabled") ||
+		d.HasChange("tag_names") {
```

Comment:
Any particular reason why this is here? We aren't doing anything with it in the `WorkspaceUpdateOptions`

Reply:
nice catch. it was in there when i first started coding this. thanks!